docs/core/deploying/ready-to-run.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Ready to Run
+description: Learn what ReadyToRun is and why you should consider using it as part of the deployment.
+author: davidwr
+ms.author: davidwr
+ms.date: 09/21/2020
+---
+# ReadyToRun Compilation
+Startup time and application latency can be improved by compiling your application assemblies as ReadyToRun (R2R) format. R2R is a form of ahead-of-time (AOT) compilation.
+
+R2R binaries improve startup performance by reducing the amount of work the just-in-time (JIT) compiler needs to do as your application loads. The binaries contain similar native code compared to what the JIT would produce. However, R2R binaries are larger because they contain both intermediate language (IL) code, which is still needed for some scenarios, and the native version of the same code. R2R is only available when you publish a self-contained app that targets specific runtime environments (RID) such as Linux x64 or Windows x64.
+
+To compile your project as ReadyToRun, do the following:
+
+01. Add the `<PublishReadyToRun>` setting to your project:
+
+    ```xml
+    <PropertyGroup>
+      <PublishReadyToRun>true</PublishReadyToRun>
+    </PropertyGroup>
+    ```
+
+02. Publish an application. For example, this command creates a app for the 64-bit version of Windows:
+
+    ```dotnetcli
+    dotnet publish -c Release -r win-x64
```

Comment:
The doc above says it is available only for self-contained apps. However, you are not passing the `--self-contained` command line option here. Does the publish automatically add that when the project has the PublishReadyToRun property set?

Reply:
Ah, oops, I meant to remove the self-contained requirement... that was changed.

Reply:
Ah, that requirement changed, and is no longer in force. I missed that when I copied and pasted from the what's new document in .NET Core 3.0